docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -1,211 +1,172 @@
-
-> **Hands on**: Select the last node of the Scene Tree view (called `RectangleArena`).
-In order to add the E-puck node, click on the `Add` button at the top of the Scene Tree view.
-In the open dialog box, and choose `PROTO (Webots) / robots / gctronic / e-puck / E-puck (Robot)`.
-Then save the simulation.
-
-<!-- -->
-
-> **Note**: Now if you run the simulation, the robot moves: that's because the robot uses a default controller with that behavior.
-Please pause and reload the world before going on.
-
-<!-- -->
-
-> **Note**: You can change the robot's position in the 3D view using the translation and rotation handles (see [this section](the-3d-window.md#axis-aligned-handles)).
-
-> Alternatively, the following keyboard shortcuts are available:
-
-> *SHIFT + left-click + drag* to move the robot parallel to the floor;
-
-> *SHIFT + mouse-wheel* to move the robot up or down.
-
-> Finally, it is possible to apply a force to the robot: *ALT + left-click + drag*.
-
-> **Note**: On Linux, you should also press the *CTRL* key in addition to *ALT + left-click + drag*.
-
-<!-- -->
-
-> **Note**: Starting the simulation by pressing the `Run` button will make Webots running the simulation as fast as possible.
-In order to obtain a real-time simulation speed, the `Real-Time` button needs to be pressed.
-
-Now we are going to modify the world and decrease the step of the physics simulation: this will increase the accuracy of the simulation.
-
-> **Hands on**: In the Scene Tree view, expand the [WorldInfo](../reference/worldinfo.md) node (the first node).
+Therefore, any modification of the world should be performed in that order: **pause, reset, modify and save the simulation**.
+
+We don't need to create the e-puck robot from scratch, we will just have to import a `E-puck` node.
+This node is actually a [PROTO](../reference/proto.md) node, like the `RectangleArena` or the `WoodenBox` we introduced before.
+
+> **Hands on #5**: Select the last node `WoodenBox` of the scene tree view.
+Click on the `Add` button (plus sign) at the top of the scene tree view.
+In the dialog box, choose `PROTO (Webots) / robots / gctronic / e-puck / E-puck (Robot)`.
+An e-puck robot should appear in the middle of the arena.
+Move and rotate this robot, the same way you did it with the boxes.
+Save the simulation and press the `Run real-time` button (right arrow).
+
+The robot should move, blink LEDs and avoid obstacles.
+That's because it has a default controller with that behavior.
+You may have noticed a small black window appearing in the upper-left corner of the 3D view.
+It shows the image taken by the [Camera](../reference/camera.md) of the e-puck robot.
+This image will remain black until the camera is explicitly enabled by the controller of the robot.
+This small image window can be moved around by dragging it.
+It can also be resized by dragging the bottom-right corner.
```

Comment:
https://cyberbotics.com/doc/guide/the-3d-window?version=fix-tutorial-1#moving-and-resizing-overlays